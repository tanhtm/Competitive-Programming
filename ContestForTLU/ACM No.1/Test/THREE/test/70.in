495951297000000000
