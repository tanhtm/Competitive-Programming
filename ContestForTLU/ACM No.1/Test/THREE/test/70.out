1 2 3 8 14 15 17 18 23 26 29 31 33 
12 13 16 21 22 24 27 28 32 35 37 
